src/components/Timestamp/styled.ts

```diff
@@ -1,9 +1,11 @@
 import { styled } from '@/styles';
 
-const Container = styled('span', {
+const Container = styled('div', {
   typo: { size: 12, height: '17px' },
   color: '$darkGrey',
   whiteSpace: 'nowrap',
+  flexShrink: 0,
+  width: 50,
```

Comment:
I wouldn't set an explicit width here, I'd do it within the `Chat` dialog since the width only makes sense within that context (or maybe `SystemResponse` like where the original marginRight was being set)

```ts
// src/components/Chat/styled.ts
import Timestamp from '@/components/Timestamp';

export const Dialog = styled('div', {
  [`& ${Timestamp.Container}`]: {
    width: 50
  }
})
```